ext/strings.go

```diff
@@ -327,6 +327,11 @@ func (stringLib) CompileOptions() []cel.EnvOption {
 					d := delim.(types.String)
 					return stringOrError(joinSeparator(l.([]string), string(d)))
 				}))),
+		cel.Function("escape", cel.MemberOverload("string_escape", []*cel.Type{cel.StringType}, cel.StringType,
```

Comment:
Let's call this `quote` since `escape` depends on the context in which you want to escape the string. Also, we should probably follow the convention for encoding that we did for `base64.encode` by making it a globally namespaced `strings.quote` since we might want to introduce `strings.unquote` at a later point as well.

Reply:
(marking as resolved, it's now renamed to `quote` and no longer a receiver function)